client/my-sites/customer-home/main.jsx

```diff
@@ -644,10 +648,15 @@ const connectHome = connect(
 		const isAtomic = isAtomicSite( state, siteId );
 		const isChecklistComplete = isSiteChecklistComplete( state, siteId );
 		const createdAt = getSiteOption( state, siteId, 'created_at' );
+		const user = getCurrentUser( state );
+		const siteIsUnlaunched = isUnlaunchedSite( state, siteId );
+		const displayWelcomeBanner =
+			siteIsUnlaunched && user.date && new Date( user.date ) < new Date( '2019-08-06' );
```

Comment:
Why the siteIsUnlaunched check here?

Reply:
```suggestion
			user.date && new Date( user.date ) < new Date( '2019-08-06' );
```

Reply:
It was to handle the case of hiding the welcome banner if we're showing the "you launched your site!" banner, but it obviously does it badly 🙃 b928c91c2e76dfc5c9af8cc31b4687a4080c5caa fixes it.